src/store/modules/principal.js

```diff
@@ -5,6 +5,20 @@ import Tab from '../../contracts/tab';
 import ResourceType from '../../constants/resourceType';
 import Resource from '../../contracts/resource';
 
+
+function sortResource (r1, r2) {
+  let res = state.statuses[r2.status.status] - state.statuses[r1.status.status]
+  if (res === 0) {
+    if (r1.vehicleType > r2.vehicleType) {
```

Comment:
I believe in case res === 0 is enough to return  r2.vehicleType - r1.vehicleType.